.github/workflows/node.js.yml

```
@@ -0,0 +1,37 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: exercism/javascript ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install project dependencies
+      run: | 
+        npm ci
+        bin/fetch-configlet
```

```suggestion


    - name: Fetch configlet
      uses: exercism/github-actions/configlet-ci@master
```

This potentially saves you work on updating these

---

Good idea. Only downside is: local thing is then no-longer necessarily the same as running it on Github Actions.

---

Upside of that: the GHA one is always the latest release whereas the local one may be outdated (if it ever changes, currently it doesn't)

---

(I more meant that the local _fetch script_ might be outdated, but alas).